src/kOS/Suffixed/Direction.cs

```diff
@@ -160,6 +160,16 @@ private void DirectionInitializeSuffixes()
             return new Vector(a.Rotation * (Vector3d)b);
         }
 
+        public static Vector operator +(Direction a, Vector b)
+        {
+            return new Vector(a.Rotation * (Vector3d)b);
+        }
+
+        public static Vector operator +(Vector b, Direction a)
+        {
+            return new Vector(a.Rotation * (Vector3d)b);
+        }
+
```

Comment:
I don't think I like this.  It's just turning the ``+`` operator into a synonym for the ``*`` operator, as far as I can tell, and that's .... sort of mathematically wrong.  You can multiply two matrices of different dimensions together, but you can't add them together.  I mean, we *can* do this.  I really don't think that we *should*.